### DISCUSSION TITLE
Can someone install this for me (paid) ?

### DISCUSSION THREAD

**Discussion by samboa86** [itsToggle/plex_debrid#640] [Q&A]
I am a total noob and don't understand cloning and all this other jargon. I see some services on Fiverr but not for this specific plex_debrid. Wondering if anyone is willing to install. Average price seems to be $10-$20 but not sure if this is more complicated than sonarr radarr jellyfin etc installs on Fiverr

**Comment by demondamz** [comment]
Please contact me I can assist you for free
